balancer/grpclb/grpc_lb_v1/load_balancer_grpc.pb.go

```diff
@@ -154,3 +154,11 @@ var LoadBalancer_ServiceDesc = grpc.ServiceDesc{
 	},
 	Metadata: "grpc/lb/v1/load_balancer.proto",
 }
+
+const (
+	LoadBalancer_BalanceLoad_FullMethod = "/grpc.lb.v1.LoadBalancer/BalanceLoad"
+)
+
+var LoadBalancer_FullMethods = []string{
+	LoadBalancer_BalanceLoad_FullMethod,
+}
```

Comment:
Is this required? If so, why?
Isn't consts for all method names not enough? (as generated in lines 158-160 of this file).

Reply:
This one is something I've used in the past to check we had policies for all endpoints but isn't strictly required, since the entire list (or even just the number of endpoints) can be obtained from the service descriptor, for example in the helloworld example as `len(Greeter_ServiceDesc.Methods)`.
Happy to remove to keep the api surface smaller.

Reply:
Latest commit removes this